src/wallet/fidelity.rs

```diff
@@ -167,6 +167,34 @@ impl Wallet {
         &self.store.fidelity_bond
     }
 
+    /// Display the fidelity bonds
+    pub fn get_fidelity_bonds_display(&self) -> Result<String, WalletError> {
+        let blockchain_info = self.rpc.get_blockchain_info()?;
+        let current_block = blockchain_info.blocks as u32;
+
+        let serialized: Vec<serde_json::Value> = self
+            .get_fidelity_bonds()
+            .iter()
+            .map(|(index, (bond, _, _))| {
+                // assuming that lock_time is always in height and never in seconds.
+                let expiring_height = bond.lock_time.to_consensus_u32();
+                let expires_in = expiring_height - current_block;
```

Comment:
nit: 
```suggestion
let expires_in = bond.lock_time.to_consensus_u32() - current_height;
```

As `expirying_height`  variable seems unnecessary -> we can directly compute `expires_in`  a single step , thus making the code more concise.